sdk/daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltinFun.scala

```diff
@@ -472,8 +493,10 @@ private[lf] object SBuiltinFun {
     }
 
   final case object SBToText extends SBuiltinPure(1) {
-    override private[speedy] def executePure(args: ArraySeq[SValue]): SText =
+    override private[speedy] def executePure(args: ArraySeq[SValue], machine: Machine[_]): SText = {
```

Comment:
I'd suggest that all builtins should have a call to potentially update the cost model. But, in this case, within the cost model we simply don't perform any update.

Reply:
Let's keep it simple, we can improve when we have something working. 
Maybe some general pattern will emerge.